app/test_main.py

```diff
@@ -1,3 +1,58 @@
+import pytest
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_age,dog_age,expect_result",
+    [
+        pytest.param(
+            0, 0, [0, 0], id="test_input_ages_equal_0"
+        ),
+        pytest.param(
+            1, 1, [0, 0], id="test_input_ages_equal_1"
+        ),
+        pytest.param(
+            14, 14, [0, 0], id="test_input_ages_equal_14"
+        ),
+        pytest.param(
+            15, 15, [1, 1], id="test_input_ages_equal_15"
+        ),
+        pytest.param(
+            23, 23, [1, 1], id="test_input_ages_equal_23"
+        ),
+        pytest.param(
+            24, 24, [2, 2], id="test_input_ages_equal_24"
+        ),
+        pytest.param(
+            27, 27, [2, 2], id="test_input_ages_equal_27"
+        ),
+        pytest.param(
+            28, 28, [3, 2], id="test_input_ages_equal_28"
+        ),
+        pytest.param(
+            29, 29, [3, 3], id="test_input_ages_equal_29"
+        )
+    ]
+)
+def test_ages(cat_age: int, dog_age: int, expect_result: list) -> None:
+    actual = get_human_age(cat_age, dog_age)
+    assert actual == expect_result
```

Comment:
Avoid using the `actual` variable here, there is no need to allocate the variable only for assertion when you can make it with function directly.